spec/cassia/requests/connect_device_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'spec_helper'
+
+RSpec.describe Cassia::Requests::ConnectDevice do
+  describe '#path' do
+    it "returns the correct API endpoint" do
+      request = described_class.new( aps: ["CC:1B:E0:E7:FE:F8","CC:1B:E0:E7:FE:F9","CC:1B:E0:E7:FE:FA"],
```

Comment:
I don't think you have to pass any parameters here since we are only calling the `path` function, this could just be
```ruby
request = described_class.new
```